### ISSUE TITLE
Accessibility: Important Card Info is Currently Not Perceivable with Screen Readers

### ISSUE THREAD

**Issue by BlindGuyNW** [octgn/OCTGN#1987]
User Impact: Blind users cannot gather important card information, as it seems to not be exposed to their screen readers.

Background: Arkham Horror LCG has "location," cards which have important info on the back/face down side. This takes the form of "Shroud," and "clue," info, and possibly other stuff depending on scenario. At the moment none of this info is readable, so, for instance, the "Study," card in the Gathering scenario is read identically whether it's flipped face-up or down. 

Expected behavior; Some indication should be provided as to whether a card is face up or down, and if there is info on one side and not the other, that should be included in the data read by the screen reader.

Note: Obviously this is just using Arkham Horror as the game I'm most familiar with, but applies more broadly.

**Comment by BenMatteson** [comment]
This is actually generally true for all users, currently the only way to see another face of a card like that is to switch to it (generally with a context menu option defined by the game), or with the newly added pop-out card preview window. Adding an indication that other faces exist for a card would probably be a good choice.
